src/pybind/mgr/dashboard/grafana.py

```diff
@@ -18,10 +19,14 @@ class GrafanaRestClient(object):
 
     @staticmethod
     def url_validation(method, path):
-        response = requests.request(
-            method,
-            path,
-            verify=Settings.GRAFANA_API_SSL_VERIFY)
+        try:
+            response = requests.request(
+                method,
+                path,
+                verify=Settings.GRAFANA_API_SSL_VERIFY)
+        except SSLError:
+            logger.exception("Failed to verify Grafana dashboard for URL %s", path)
```

Comment:
If the exception contains useful information, then i would suggest to log this, too.

Reply:
The exception is logged but not returned to the front-end. See the [docs](https://docs.python.org/2.7/library/logging.html#logging.Logger.exception).

Reply:
I meant to log something like
``` 
except SSLError as e:
            logger.exception("Failed to verify Grafana dashboard for URL %s: %s", path, str(e))
``` 
In case the SSLError contains useful information that might help the user to udentify the SSL problem. If the exception only contains blabla then it's worthless to log it.

Reply:
I'm sorry, I should probably have been more precise. The [documentation](https://docs.python.org/2.7/library/logging.html#logging.Logger.exception) for the `exception` method says:

> Exception info is always added to the logging message. This method should only be called from an exception handler.

There is no need to add the exception to the log message, it will automatically be appended. This is the use case the `exception` method has been created for.